packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -178,6 +178,12 @@ export type AddMediaOptions = {
   allowMediaInLobby?: boolean; // allows adding media when in the lobby
 };
 
+export type CallStateForMetrics = {
+  correlationId?: string;
+  joinTrigger?: string;
+  loginType?: string;
```

Comment:
could be typed

Reply:
I know, nothing else seemed to be so I didn't either, I was lazy here and strict in cantina